### ISSUE TITLE
Flair doesn't work correctly with CPU optimised torch for linux

### ISSUE THREAD

**Issue by kislerdm** [flairNLP/flair#1528]
**Description**

Error when calling the method **_apply** of the class **flair.models.language_model.LanguageModel**:

```bash
/usr/local/lib/python3.8/site-packages/flair/models/language_model.py in <genexpr>(.0)
    391 
    392     def _apply(self, fn):
--> 393         major, minor, build, *_ = (int(info)
    394                                 for info in torch.__version__.split('.'))
    395 

ValueError: invalid literal for int() with base 10: '0+cpu'
```

It is blocking functionality of the library when it comes to *embeddings*, or *language_model* definition.

**Error cause**

Error is caused by parsing torch version (see the line *394* of the module *flair.models.language_model.py*). It fails when a CPU specific torch version is being used. For example, the CPU optimised version of torch-1.4.0 for linux (available [here](https://download.pytorch.org/whl/cpu/torch-1.4.0%2Bcpu-cp38-cp38-linux_x86_64.whl)) has `__versions__ = '1.4.0+cpu'`.

**Solution proposal**

*Status quo*:
```python
def _apply(self, fn):
        major, minor, build, *_ = (int(info)
                                for info in torch.__version__.split('.'))
```

*Fix*:
```python
def _apply(self, fn):
        major, minor, *_ = (int(info) for info in torch.__version__.split('.')[:-1])
```

The object `build` unpacked from the `torch.__version__` parsing result's tuple is not being used, ergo its removal would be a reasonable solution.

**Environment:**
 - OS: linux debian 10 (base of the docker image `python:3.8.1-slim-buster`)
 - torch==1.4.0+cpu
 - flair==0.4.5

**Instruction to reproduce**

Build the image, it will fail due described error.

```Dockerfile
FROM python:3.8.1-slim-buster AS base

WORKDIR /

RUN apt-get update -y \
  && pip install --upgrade pip \
  && pip install --no-cache-dir "https://download.pytorch.org/whl/cpu/torch-1.4.0%2Bcpu-cp38-cp38-linux_x86_64.whl" flair==0.4.5

RUN python -c "import flair; from flair.embeddings import StackedEmbeddings, FlairEmbeddings;\
embeddings = StackedEmbeddings([FlairEmbeddings('mix-forward')]);
```

**Comment by alanakbik** [comment]
Hello @kislerdm sorry for the late reply. This issue was fixed in #1388 and will be part of the next Flair release. In the meantime, you can install with: 
```console
pip install --upgrade git+https://github.com/flairNLP/flair.git
```
That should fix it!

**Comment by alanakbik** [comment]
Fixed in the new release of Flair.
